project/plugin.sbt

```diff
@@ -5,3 +5,5 @@ addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.0")
 addSbtPlugin("com.eed3si9n" % "sbt-unidoc" % "0.4.1")
 
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.10")
+
+addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "5.2.4")
```

Comment:
I remember that for some time, the sbteclipse-plugin was breaking things. that's why i removed it.
Isn't the scala eclipse support dead / not realy active ?

Reply:
I'm currently developing Spinal in Eclipse. It's not great, but it works and I didn't encounter anything breaking.

Reply:
I had bad experiances with eclipse scala plugin.
Did you tried the intelij comunity edition with the scala plugin ?

Reply:
@Dolu1990 No, I don't have IntelliJ installed

Reply:
If you have time, try intellij, hard to go back to eclipse after it :)
(triple shift magic)